tests/includes/class-test-move.php

```diff
@@ -78,7 +78,7 @@ public function test_account_with_invalid_user() {
 	 */
 	public function test_account_with_invalid_target() {
 		$from = Actors::get_by_id( self::$user_id )->get_id();
-		$to   = 'https://invalid-url.com/user/1';
+		$to   = 'https://example.com/user/1';
```

Comment:
I kept getting `wp_validate_url()` errors when running tests locally, not sure why?